### ISSUE TITLE
FEATURE: Front-End Form Submissions Display & CSV Download Link

### ISSUE THREAD

**Issue by brandbrilliance** [Desertsnowman/Caldera-Forms#232]
Features:

![caldera-forms-export](https://cloud.githubusercontent.com/assets/1362479/9541019/8d8c6e3e-4d64-11e5-9eb6-b282cebfe9ce.jpg)

* You can display submissions in a (hardcoded) table in the front-end using a shortcode
* Optional download link (with class) support to make a link/button that is only valid for that DAY, as the download link is md5 encoded and checked to match all forms and current day and if not, doesn't do anything (data spread safety precaution).
* Intention is that you can place form submissions shortcode on a password protected page (easier to control access) and provide entries to clients/organisers/others, etc.

I've coded this as a separate plugin item, for easier testing, but you're welcome to incorporate the code into the base. I'm using it for personal event registration management usage.

[Download as Plugin](https://dl.dropboxusercontent.com/u/28931329/_code/caldera-forms-export.zip)

**Comment by Desertsnowman** [comment]
That is great.

We're actually working on a Caldera Forms View plugin that will allow you to create data views for the front end just like you showing here, but with more templating, filtering, editing etc..

**Comment by Desertsnowman** [comment]
Have you thought about releasing the plugin on wordpress.org?

**Comment by brandbrilliance** [comment]
Nope. I'm not really a coder. This was just my best effort hack to solve a problem. Releasing it officially means endlessly supporting it. I've got one plugin and that is already enough for me.

**Comment by dlincoln** [comment]
Nice work man! Glad to hear it's slated for an "official" implementation, but this looks like a more useful view. We have some chores this would be terrific for so I'm giving it a try.

You know, I'm not sure you SHOULD have entry editing conceptually. An "entry" is presumably created by a "user", and sometimes represents a legally binding document. In those cases staff editing of "canonical" content is inappropriate - better to ingest/apply the content as far as possible, contact the user and get them to resubmit in a legally binding context, or to agree in email to letting you adjust the parameters they've set (giving you a paper trail in the event they sue). When and where the data is deployed and used, you then correct it in the database. You can leave the "flawed" entry around or delete it at that point.

But yeah, I'd like the feature too. So my cases would be better covered by adding an option to make entries non-editable in general settings of the form.

**Comment by zoomwebstudio** [comment]
That is awesome..


Cordialement et avec mes salutations distinguées,

K. N. Adevou Fera EKPEH
*"Innovation distinguishes between a leader and a follower." - Steve Jobs*

Webdesigner - WordPress
Spécialiste des sciences de l'Information documentaire
Consultant web (Design & Social Media)
CEO @ CoucouMail Web & Services <http://www.coucoumail.com>

Phone: +228 92 14 65 91 (WhatsApp | Viber | Telegram)

Skype: adevou
*#SiliconMind*
------------------------------
Behance: http://www.behance.net/adevouferaekpeh
Facebook: http://www.facebook.com/lordfera
LinkedIn: https://www.linkedin.com/in/adevoufera
Pinterest : http://www.pinterest.com/lordfera
Twitter: https://www.twitter.com/adevlordfera



2015-08-28 11:49 GMT+00:00 dlincoln <notifications@github.com>:

**Comment by zoomwebstudio** [comment]
Hi Desertsnowman/Calder,

Adevou Fera K. N. EKPEH wants to connect with you on LinkedIn.
Join now to view their profile and start your own.

Confirm that you know Adevou Fera K. N.: https://www.linkedin.com/e/v2?e=0-iocjz3gv-zv&t=ssuw&tracking=eml-guest-invite-cta&ek=invite_guest&sharedKey=PXYArjHW&invitationId=6138602312542797825








You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=0-iocjz3gv-zv&t=lun&midToken=AQFmrFVhGEd36Q&ek=invite_guest&loid=AQGhImNZU679DgAAAVTCyazYlL9dBCaeM4L23WDbaYakyHnDX0NbB88lyRzVMGSjvHG3f--ena3GJZVFhlNLkrbUcOHh1thZvXlzhCtytWs5kwYA4H3fAwio57fiWtQxHxJa9fp6ld6yAsUyTlcK2zWzxbZKBy8IxvAafVTV4BBxIUK1-2lGTCkF-jCZEct1Berk&eid=0-iocjz3gv-zv


This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=0-iocjz3gv-zv&a=customerServiceUrl&ek=invite_guest

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.

**Comment by zoomwebstudio** [comment]
Adevou Fera K. N. EKPEH would like to connect on LinkedIn. How would you like to respond?

Accept: https://www.linkedin.com/e/v2?e=-naizsp-iomlpv0t-tb&t=ssuw&ek=first_guest_reminder_01&li=14&m=hero&ts=accept_text&sharedKey=PXYArjHW&invitationId=6138602312542797825

View Adevou Fera K. N. EKPEH&#39;s profile: https://www.linkedin.com/e/v2?e=-naizsp-iomlpv0t-tb&t=ssuw&ek=first_guest_reminder_01&li=3&m=hero&ts=profile_text&sharedKey=PXYArjHW&invitationId=6138602312542797825

Desertsnowman/Calder,

J’aimerais vous inviter à rejoindre mon réseau professionnel sur LinkedIn.





You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=-naizsp-iomlpv0t-tb&t=lun&midToken=AQFmrFVhGEd36Q&ek=first_guest_reminder_01&li=16&m=unsub&ts=HTML&eid=-naizsp-iomlpv0t-tb&loid=AQHPcqzewPZRoAAAAVToOAtBpolXo-DM6Cp8Sr5nHeDu-9osH5rHWtZYmvy4FixJGt_-NPj2vWnA9UBv3SPTZ4kUkcNzQRXqa1SJ27GQBfGIgYSVtApXV1-HomtWyk6laB2p1C8VtFi4Nnz_hMGUxUZokBZQgrQ2dhPct5co0N2opz1aFuhbyTiuUMWcxKDuNj7c

This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=-naizsp-iomlpv0t-tb&a=customerServiceUrl&ek=first_guest_reminder_01

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.

**Comment by zoomwebstudio** [comment]
Adevou Fera K. N. EKPEH would like to connect on LinkedIn. How would you like to respond?

Accept: https://www.linkedin.com/comm/start/accept-invitation?sharedKey=PXYArjHW&invitationId=6138602312542797825&trk=eml-second_guest_reminder_01-hero-13-accept_text&trkEmail=eml-second_guest_reminder_01-hero-13-accept_text-null-%7Enaizsp%7Eiowujhm2%7Ear

View Adevou Fera K. N. EKPEH&#39;s profile: https://www.linkedin.com/comm/start/accept-invitation?sharedKey=PXYArjHW&invitationId=6138602312542797825&trk=eml-second_guest_reminder_01-hero-3-profile_text&trkEmail=eml-second_guest_reminder_01-hero-3-profile_text-null-%7Enaizsp%7Eiowujhm2%7Ear

Desertsnowman/Calder,

J’aimerais vous inviter à rejoindre mon réseau professionnel sur LinkedIn.





You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=-naizsp-iowujhm2-ar&t=lun&midToken=AQFmrFVhGEd36Q&ek=second_guest_reminder_01&li=15&m=unsub&ts=HTML&eid=-naizsp-iowujhm2-ar&loid=AQHdZ4rqyz-WnQAAAVUMswg9rFz9BhvdCvAyuFelViFy0nTzjxm-yFxgxrA1rVZMy2MRpvuXfKeGamnGt1stB2th-oxXfIL322cU9wub3c4FlQXlqtS7E5FqKRCnx2OXFxb5ACG5J_Y7zN687omtW_qoM78xv3z4E0ZbRwkz7IOoSVHBOiNfOnpnZ5hR2WTZlD-J

This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=-naizsp-iowujhm2-ar&a=customerServiceUrl&ek=second_guest_reminder_01

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.

**Comment by rawadm** [comment]
Hello I just downloaded this plugin however I would need to know how to display the short-code of the Front-End Form Submissions. Help would be appreciated.

**Comment by ander2300** [comment]
Please, upload again the plugin. Thanks

**Comment by Allison101** [comment]
Hi,
This one looks great and seems like what i really need right now.
Would really appreciate it if you could re upload the plugin again. Thanks
